app/src/main/java/com/example/pricingpal/model/CSVReader.kt

```diff
@@ -0,0 +1,80 @@
+package com.example.pricingpal.model
+
+import android.content.Context
+import java.io.FileNotFoundException
+import java.io.IOException
+import java.io.InputStream
+import java.io.InputStreamReader
+import java.util.Scanner
+
+/**
+ * Class: CSVReader
+ * @author Julian Ellis
+ * @version 2.1
+ * @written 9/24/2023
+ * This class extends AppCompatActivity and holds a function to read a CSV file
```

Comment:
This comment block needs to be updated to reflect your most recent changes.

Reply:
Made the changes. Pushing that now

Reply:
"MainActivity.kt" is misspelled if you'd like to correct that. Other than that, the updated comment block looks good!